runtime/android/core_internal/src/org/xwalk/core/internal/XWalkViewDelegate.java

```diff
@@ -248,8 +248,14 @@ public InputStream openRawResource(String resource) {
                     String resourceName = resource.split("\\.")[0];
                     int resourceId = getResourceId(context, resourceName, "raw");
                     try {
-                        return context.getResources().openRawResource(resourceId);
-                    } catch (NotFoundException e) {
+                        try {
+                            return context.getResources().openRawResource(resourceId);
+                        } catch (NotFoundException e) {
```

Comment:
It's better to have a way to check if it's download mode like `isSharedMode`. But current implementation is okay